corehq/apps/case_importer/tracking/views.py

```diff
@@ -32,6 +34,27 @@ def case_uploads(request, domain):
 
 
 @require_can_edit_data
+def update_case_upload_comment(request, domain, upload_id):
+    comment = request.POST.get('comment')
+    if comment is None:
+        return HttpResponseBadRequest("POST body must contain non-null comment property")
+    elif len(comment) > MAX_COMMENT_LENGTH:
+        return HttpResponseBadRequest("comment must be shorter than {} characters"
+                                      .format(MAX_COMMENT_LENGTH))
```

Comment:
Nit: I think these above 5 lines should go after the permissions check, although it doesn't matter much, since they don't persist anything until after.